tensorflow-core/tensorflow-core-api/src/gen/java/org/tensorflow/op/collective/BroadcastRecv.java

```diff
@@ -17,16 +17,17 @@
 
 package org.tensorflow.op.collective;
 
-import org.tensorflow.DataType;
 import org.tensorflow.Operand;
 import org.tensorflow.Operation;
 import org.tensorflow.OperationBuilder;
 import org.tensorflow.Output;
 import org.tensorflow.ndarray.Shape;
+import org.tensorflow.op.Operands;
 import org.tensorflow.op.RawOp;
 import org.tensorflow.op.Scope;
 import org.tensorflow.op.annotation.Endpoint;
 import org.tensorflow.op.annotation.Operator;
+import org.tensorflow.proto.framework.DataType;
```

Comment:
It is, it was incorporated with my last changes and I've found a way to remove it, thanks for catching it. Now the process of [collecting the imported classes](https://github.com/tensorflow/java/blob/3f3f384517a5ba633c3b1971651cdcdec3eebfa7/tensorflow-core/tensorflow-core-api/src/bazel/op_generator/op_generator.cc#L78) in the op generator is a bit cumbersome and should be reviewed at some point, I agree. It will be even more necessary when we will (finally) activate lint checks that will report unused imports as an error.